server.go

```diff
@@ -2,23 +2,47 @@ package lookout
 
 import (
 	"context"
+	"fmt"
 	"sync"
 
+	"github.com/src-d/lookout/pb"
 	log "gopkg.in/src-d/go-log.v1"
+	yaml "gopkg.in/yaml.v2"
 )
 
-type reqSent func(AnalyzerClient) ([]*Comment, error)
+// AnalyzerConfig is a configuration of analyzer
+type AnalyzerConfig struct {
+	Name string
+	// Addr is gRPC URL.
+	// can be defined only in global config, repository-scoped configuration is ignored
+	Addr string
+	// Settings any configuration for an analyzer
+	Settings map[string]interface{}
+}
+
+// Config is a server configuration
+type Config struct {
```

Comment:
a name like `ServerConfig` may be could help avoid the confusion?

Reply:
renamed